public/css/style.css

```diff
@@ -515,10 +515,28 @@
     position: absolute;
     top: 0;
     left: 52px;
-    width: 125px;
-    height: 50px;
     z-index: 10;
+  }
```

Comment:
The [Tahoe app uses sass,](https://github.com/BlinkTagInc/tahoebikemap/tree/master/public/scss) but it seems like this app just uses raw css, directly edited in this file. Am I getting that right?